io.catenax.generic.digital_product_passport/2.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1142 @@
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Aluminium" .
+
+:recycled a samm:Property ;
+   samm:preferredName "Recycled"@en ;
+   samm:description "A flag if the material is recovered recycled content from the product. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Article 1:\n(1) (f) recycled content in products."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue false .
+
+:renewable a samm:Property ;
+   samm:preferredName "Renewable"@en ;
+   samm:description "A flag if the material is from a renewable resource that can be replenished. Renewable resources are those that can be reproduced by physical, chemical, or mechanical processes. These are the kind of resources that can be regenerated throughout time. Forest wood, for example, can be grown through reforestation. This attribute is complemented to the ESPR proposal by the amendments of the European Parliament from July 12th, 2023 Annex I:\n(ha) use or content of sustainably sourced renewable materials."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:materialValue a samm:Property ;
+   samm:preferredName "Material Value"@en ;
+   samm:description "Concentration of the material in correlation to the material sum."@en ;
+   samm:characteristic :ActualConcentration ;
+   samm:exampleValue "5.0"^^xsd:float .
+
+:materialIdentification a samm:Property ;
+   samm:preferredName "Material Identification"@en ;
+   samm:description "The material identification, in accordance with the specification in the attribute list type."@en ;
+   samm:characteristic :IdentifierCharacteristic ;
+   samm:exampleValue "CAS" .
+
+:environmentalFootprint a samm:Property ;
+   samm:preferredName "Environmental Footprint"@en ;
+   samm:description "The environmental footprint of the product. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Annex I:\n(l) the environmental footprint of the product, expressed as a quantification, in accordance with the applicable delegated act, of a product?s life cycle environmental impacts, whether in relation to one or more environmental impact categories or an aggregated set of impact categories."@en ;
+   samm:characteristic :FootprintList .
+
+:carbonFootprint a samm:Property ;
+   samm:preferredName "Carbon Footprint"@en ;
+   samm:description "The carbon footprint of the product. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Annex I:\n(m) the carbon footprint of the product."@en ;
+   samm:characteristic :FootprintList .
+
+:lifeUnit a samm:Property ;
+   samm:preferredName "Life Unit"@en ;
+   samm:description "The unit of the respective lifespan expressed through the possible units day, month, cycle, year and runningOrOperatingHour."@en ;
+   samm:see <https://eclipse-esmf.github.io/samm-specification/snapshot/appendix/unitcatalog.html> ;
+   samm:characteristic :LifeEnumeration ;
+   samm:exampleValue "unit:month"^^samm:curie .
```

Comment:
this is currently not validating. I am investigating why

Reply:
changed back to string